pkgs/build-support/dotnet/build-dotnet-module/hooks/dotnet-configure-hook.sh

```diff
@@ -43,7 +54,7 @@ EOF
     find -name paket.dependencies -exec sed -i 's+source .*+source @nugetSource@/lib+g' {} \;
     find -name paket.lock -exec sed -i 's+remote:.*+remote: @nugetSource@/lib+g' {} \;
 
-    dotnet tool restore --add-source "@nugetSource@/lib"
+    #dotnet tool restore --add-source "@nugetSource@/lib"
```

Comment:
Commented out due to making a network call; I believe that telemetry will just need to be disabled with `export DOTNET_CLI_TELEMETRY_OPTOUT=1` now that I think about it.